responsible-ai-collaborative/aiid
# Add onResendConfirmationClick function on the toast message

This closes https://github.com/responsible-ai-collaborative/aiid/issues/2335

## Steps to test
- Signup with a new user
- You should receive the account verification email
- Try to login with that user without confirm the account via email
- A toast message appears with the option to "Resend Verification email", click it
- You should receive another account verification email
